tests/func/gdrive-settings.yaml

```diff
@@ -0,0 +1,16 @@
+client_config_backend: settings
+client_config:
+    client_id: 719861249063-v4an78j9grdtuuuqg3lnm0sugna6v3lh.apps.googleusercontent.com
```

Comment:
does it mean that anyone has access to our testing gdrive?

Reply:
It's oauth2 credentials to utilize specific `Google Project`'s API. Supposed to be used only by DVC's CI tools, but, yes, anyone can grab it, use to exceed API usage limits and affect stability of ci process.

Reply:
do we use some GDrive account to run tests? does this mean, for example, that one can take these credentials and upload something to that account?

Reply:
> do we use some GDrive account to run tests?

GDrive (Google) account should be created to be used for tests on ci.  `test_data_cloud` tests are skipped till `GDRIVE_USER_CREDENTIALS_DATA` env variable is not set.

> does this mean, for example, that one can take these credentials and upload something to that account?

no, these credentials is to access Google API, but not specific user's storage. Please see more details in answer on related question https://github.com/iterative/dvc/pull/2551#discussion_r339329063